inductiva/resources/machines.py

```diff
@@ -0,0 +1,112 @@
+"""Containes MachineGroup class to manage GC resources."""
+from absl import logging
+from typing import Optional
+import uuid
+import inductiva
+from inductiva import api
+from inductiva import resources
+
+from inductiva.client import ApiClient, ApiException
+from inductiva.client.apis.tags.instance_api import InstanceApi
+from inductiva.client.model.instance import Instance
+from inductiva.client.model.instance_group import InstanceGroup
+
+
+class MachineGroup():
+    """Class to manage Google Cloud resources."""
+
+    def __init__(
+        self,
+        machine_type: str,
+        num_machines: int = 1,
+        spot: bool = False,
+        disk_size_gb: int = 30,
+        label: str = None,
+        machine_group_id: Optional[uuid.UUID] = None,
+    ) -> None:
+        """Create a MachineGroup object.
+
+        Args:
+            machine_type: The type of GC machine to launch. Ex: "e2-medium".
+            num_machines: The number of machines to launch.
+            spot: Whether to use spot instances.
+            disk_size_gb: The size of the disk in GB. (min. 30 GB)
```

Comment:
I don't think this minimum is enforced, so maybe it should say something like "recommended minimum of 30 GB"?

Reply:
I see. I'm just saying the docstring shouldn't say "min. 30 GB" if that minimum isn't enforced somewhere else in the code. Say instead that you recommend a minimum of 30 GB.